### PR TITLE
Adds glitter bombs

### PR DESCRIPTION
**Pull request by lordpidey**
:cl: lordpidey
add: Glitter bombs have been added to arcade prizes.
/:cl:

So, this PR adds glitter bombs, which are just grenades that leave behind decals that LOOK like a gas leak.  Glitter can be cleaned through anything that cleans normal messes.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
no recipe? :-1: fix this

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1528,3 +1528,37 @@
...
+/datum/reagent/glitter/white
+	name = "white glitter"
+	id = "white_glitter"
+	description = "white sparkles that get everywhere"
+	glitter_type = /obj/effect/decal/cleanable/glitter/white
+
+/datum/reagent/glitter/blue
+	name = "blue glitter"
+	id = "blue_glitter"
+	description = "pink sparkles that get everywhere"
```
Comment (swindly) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: whoops

**Comment by lordpidey** [comment]
goof, it might be too powerful if there's a chem recipe for this.

**Comment by ChangelingRain** [comment]
These should probably not animate or have some really obvious visual difference from actual gas leaks

**Comment by Armhulen** [comment]
but what's the fun in that?

**Comment by Iamgoofball** [comment]
@ChangelingRain why add a mechanic for faking a gas leak and then not make it actually fake a gas leak

**Comment by Armhulen** [comment]
https://tgstation13.org/wiki/Fun read the rules goofball

**Comment by optimumtact** [comment]
merge conflicted fix and I'll merge

**Pull request merged**